Resources/Private/Fusion/Pagination.fusion

```diff
@@ -31,22 +33,25 @@ prototype(Flowpack.Listable:Pagination) < prototype(Neos.Fusion:Component) {
         itemRenderer = Neos.Fusion:Case {
             separator {
                 condition = ${i == '...'}
-                renderer = ${'<li class="' + props.itemClass + '">' + i + '</li>'}
+                renderer = ${'<li class="' + props.itemClass + '"><a class="' + props.linkClass + '">' + i + '</a></li>'}
```

Comment:
Sorry, missed this one as well.
Is it valid to have <a> tag with no `href`?

Reply:
It is valid without href, yes (since HTML5). But as the world is moving from Bootstrap to Tailwind, we might have to rethink how we render the list items (looking into the direction of Fusion-components).